dsutil/zip.go

```diff
@@ -9,26 +9,40 @@ import (
 	"io/ioutil"
 	"strings"
 
+	"github.com/ghodss/yaml"
 	datastore "github.com/ipfs/go-datastore"
 	"github.com/qri-io/cafs"
 	"github.com/qri-io/dataset"
 	"github.com/qri-io/dataset/dsfs"
 )
 
 // WriteZipArchive generates a zip archive of a dataset and writes it to w
-func WriteZipArchive(store cafs.Filestore, ds *dataset.Dataset, ref string, w io.Writer) error {
+func WriteZipArchive(store cafs.Filestore, ds *dataset.Dataset, format string, ref string, w io.Writer) error {
```

Comment:
I think we're approaching the point where this function's configuration params should be combined into a struct.  `format` and `ref` should be optional with sensible defaults.

But before we do that it'd make sense to try to formalize our approach, I think this is mergable as-is